playbooks/roles/oraclejdk/tasks/main.yml

```diff
@@ -21,6 +21,18 @@
     url: "{{ oraclejdk_url }}"
     headers: 'Cookie:oraclelicense=accept-securebackup-cookie'
     dest: "/var/tmp/{{ oraclejdk_file }}"
+  when: USE_PRIVATE_ORACLEJDK is undefined or not USE_PRIVATE_ORACLEJDK
```

Comment:
@arjunsinghy96, `oraclejdk_url` is already an ansible variable that can have any value including private links. So can you use a different name for this variable that clearly indicates what it is used for?

Reply:
@lgp171188 The variable `oraclejdk_url` is not touched in this PR and neither is its value used in the new task.
I have used the new variable `USE_PRIVATE_ORACLEJDK` to conditionally skip this task if it is defined and set to `True`

Reply:
@arjunsinghy96, what I meant here is that even `oraclejdk_url` can point to a private URL and hence suggested renaming this to something more specific and relevant.